pkg/controllers/job/job_controller_actions.go

```diff
@@ -489,6 +498,16 @@ func (cc *jobcontroller) createOrUpdatePodGroup(job *batch.Job) error {
 				job.Namespace, job.Name, err)
 			return err
 		}
+
+		minTaskMember := map[string]int32{}
+		for _, task := range job.Spec.Tasks {
+			if task.MinAvailable != nil {
+				minTaskMember[task.Name] = *task.MinAvailable
+			} else {
+				minTaskMember[task.Name] = task.Replicas
```

Comment:
Will it conflict with `Job.Minabailable = 1`

Reply:
What do you mean by `conflict` here, can you elaborate more? Normally, the task.minavailable here should not be empty because it is filled in the webhook

Reply:
```
Job:
  minAvailable: 1
  tasks:
  - task1
       replicas: 10
  - task2
       replicas: 3
```

If I specify the `vj` which task.minavailable is omitted, the original logic job will be scheduled if one pod minresouce meet.
In this pr, job will never be scheduled until total 13 pods resources can be allocated, right?

Reply:
No, I have judged  sumof(task minavailable) and job.minavailable, if job.minavailable<sumof(task minavailable), we will ignore task minavailable.
See:
```
// CheckTaskMinAvailable returns whether each task of job is valid.
func (ji *JobInfo) CheckTaskMinAvailable() bool {
	// if job minAvailable is less than sumof(task minAvailable), skip this check
	if ji.MinAvailable < ji.TaskMinAvailableTotal {
		return true
	}
        ...
}
```